modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/4_claim.json

```
@@ -583,6 +583,96 @@
+			"externalReferenceCode": "RAYCLAIM-078",
+			"incidentDate": "2023-01-09",
+			"r_policyToClaims_c_raylifePolicyERC": "PO-10-358-3844",
+			"settledDate": null
+		},
+		{
+			"claimAmount": 373,
+			"claimCreateDate": "2022-09-12",
+			"claimStatus": "approved",
+			"dataJSON": "{\"Detail\":{\"InsuranceInfo\":{\"InsuredName\":\"Lothario Fantonetti\",\"InsuredSSN\":\"578-33-0494\",\"InsuredVehicle\":\"2013 Ford Mustang GT Fastback\",\"InsuredVIN\":\"3D7JV1EP5AG586540\"},\"IncidentDetail\":{\"Location\":\"245 Fordem Crossing\",\"LocationCity\":\"San Jose\",\"LocationState\":\"CA\",\"LocationZIP\":\"95113\",\"Type\":\"Vehicle incident while parked\",\"Detail\":\"Incident with more than one passenger\",\"OtherVehicleMake\":\"Mercedes-Benz\",\"OtherVehicleModel\":\"C-Class\",\"OtherVehicleVIN\":\"WAUMF98PX6A901480\",\"OtherVehicleOccupant\":\"Robbi Dillingston\",\"OtherVehicleOccupant2\":\"Forrester Brunton\",\"OtherVehicleOccupant3\":\"Beatrix Mantrup\",\"OtherVehicleInsurance\":\"Liberty Mutual\",\"Damage\":{\"Condition\":\"Minor damage and safe to drive\",\"Interior\":\"Seats\",\"Exterior\":\"Front,Driver Side\"}},\"VehicleRepair\":{\"Status\":\"Pending\",\"Description\":\"Vehicle pending repair\",\"MechanicName\":\"Lyons Auto Pro Plus\",\"MechanicPhone\":\"(310) 987-4567\",\"MechanicAddress\":\"675 Main Street Los Angeles, CA 90005\",\"MechanicOrderNum\":\"iry2kw\",\"MechanicCompleteDate\":\"2022-09-25T19:20:31Z\",\"MechanicCost\":4077},\"Activity\":{\"Date1\":\"2022-09-07T19:20:31Z\",\"Desc1\":\"Claim Submitted\",\"By1\":\"Scott Producer (Producer)\",\"Date2\":\"2022-09-09T19:20:31Z\",\"Desc2\":\"Investigation in progress\",\"By2\":\"Jack Johnson (Claim Adjuster)\",\"Date3\":\"2022-09-14T19:20:31Z\",\"Desc3\":\"Estimation complete\",\"By3\":\"Jack Johnson (Claim Adjuster)\",\"Date4\":\"2022-09-25T19:20:31Z\",\"Desc4\":\"Vehicle Repair Complete\",\"By4\":\"Scott Producer (Producer)\"}}}",
+			"externalReferenceCode": "RAYCLAIM-079",
+			"incidentDate": "2022-07-10",
+			"r_policyToClaims_c_raylifePolicyERC": "PO-12-528-6979",
+			"settledDate": null
+		},
+		{
+			"claimAmount": 2510,
+			"claimCreateDate": "2022-051-27",
```

Hi Raquel, I believe this date is wrong. Maybe the month is 05?
```suggestion
			"claimCreateDate": "2022-05-27",
```